src/components/attendee/Input/Input.tsx

```diff
@@ -34,25 +36,33 @@ export const Input: React.ForwardRefRenderFunction<
   border = "borderless",
   ...extraInputProps
 }) => {
-  const inputClassNames = classNames(inputClassName, CN[border]);
+  const inputClassNames = classNames(inputClassName, CN[border], {
+    [CN.invalid]: isDefined(error),
```

Comment:
Same comment for empty string as the other place: is empty string (or other falsy value) supposed to add the `CN.invalid` class just because it isn't `null` nor `undefined`?

What happens if `error` is `false` or `NaN` or `0` ?

A comment at the end of the line might make the decision more clear.

Reply:
yep, done
[a58b99b](https://github.com/sparkletown/sparkle/pull/2905/commits/a58b99bf83fb6ed1a9b39ec673fb2882dab9e8e9)